utils.py

```diff
@@ -100,3 +103,21 @@ def import_image(path):
     image.paste(image_)
     image = np.asarray(image)
     return image
+
+
+def initialize_model(file_path):
+
+    if os.path.exists(file_path)==False:
+        checkpoint_split = file_path.split("/")[2]
```

Comment:
Windows'ta calismayacak bu kisim.

Reply:
Ben zaten windows yapmıştım bunu ama sen ne önerirsin?

Reply:
eger windows olursa double backslash olacak path icinde, ya da olabilir. Belki front slash veya backslash var mi diye cek edip ona gore split edebiliriz. Ornegin
split_key = "/" if "/" in file_path else "\\"
checkpoint_split = file_path.split(split_key )[2]